platform/viewer/src/index.js

```diff
@@ -25,18 +25,26 @@ import OHIFMeasurementTrackingExtension from '@ohif/extension-measurement-tracki
 import OHIFDICOMSRExtension from '@ohif/extension-dicom-sr';
 
 /** Combine our appConfiguration and "baked-in" extensions */
-const appProps = {
-  config: window ? window.config : {},
-  defaultExtensions: [
-    OHIFDefaultExtension,
-    OHIFCornerstoneExtension,
-    OHIFMeasurementTrackingExtension,
-    OHIFDICOMSRExtension,
-  ],
-};
+// const appProps = {
+//   config: window ? window.config : {},
+//   defaultExtensions: [
+//     OHIFDefaultExtension,
+//     OHIFCornerstoneExtension,
+//     OHIFMeasurementTrackingExtension,
+//     OHIFDICOMSRExtension,
+//   ],
+// };
 
 /** Create App */
-const app = React.createElement(App, appProps, null);
+// const app = React.createElement(App, appProps, null);
 
 /** Render */
-ReactDOM.render(app, document.getElementById('root'));
+// ReactDOM.render(app, document.getElementById('root'));
```

Comment:
Can we delete instead of comment out?

Reply:
Yes, we can delete them

Reply:
But I think we should leave it to provide faster dev speed.

Reply:
Why do the commented lines have faster dev speed? Are they used as some sort of reference or documentation?